test/system/helpers.bash

```diff
@@ -381,6 +384,28 @@ function run_podman() {
             die "exit code is $status; expected $expected_rc"
         fi
     fi
+
+    # Check for "level=<unexpected>" in output, because a successful command
+    # should never issue unwanted warnings or errors. The "0+w" convention
+    # (see top of function) allows our caller to indicate that warnings are
+    # expected, e.g., "podman stop" without -t0.
+    if [[ $status -eq 0 ]]; then
+        # FIXME: don't do this on Debian: runc is way, way too flaky:
+        # FIXME: #11784 - lstat /sys/fs/.../*.scope: ENOENT
+        # FIXME: #11785 - cannot toggle freezer: cgroups not configured
+        if [[ ! "${DISTRO_NV}" =~ debian ]]; then
+            # FIXME: All kube commands emit unpredictable errors:
+            #    "Storage for container <X> has been removed"
+            #    "no container with ID <X> found in database"
+            # These are level=error but we still get exit-status 0.
+            # Just skip all kube commands completely
```

Comment:
This, though, really bothers me. `podman kube` is noisy, scary-noisy. I would really like us to consider fixing that.